radicle-cob/src/object.rs

```diff
@@ -29,12 +30,26 @@ pub enum ParseObjectId {
 #[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub struct ObjectId(Oid);
 
+pub fn decode_hex(s: &str) -> Result<Vec<u8>, ParseIntError> {
```

Comment:
I wouldn't expose this method publicly here, it's not really part of the API. Maybe move it inside `from_str`?